ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
so in summary, you want APP_USER_LOCAL enabled so the ext4 mounted owner/group is that of your host. I think it might be dangerous doing that by default as if volume mounts are sometimes used in non-development environments (not really us but others). APP_USER_LOCAL can be set in the docker-compose.yml as false, but your host bash profile does `export APP_USER_LOCAL=true` to enable it for yourself only.